inc/gravity-forms.php

```diff
@@ -38,7 +38,7 @@ function ( $form_string, $form ) {
 				// Replace main gfield_label elements with nhsuk-label.
 				$form_string = preg_replace( '#gfield_label#s', 'nhsuk-label', $form_string );
 				// Remove <ul>s around elements.
-				$form_string = preg_replace( "#<ul class='gfield(.*?)>(.*?)</ul >#", '$2', $form_string );
+				$form_string = preg_replace( "#<ul class='gfield(.*?)>(.*?)</ul>#s", '$2', $form_string );
```

Comment:
Not sure how that affects the forms but it removes the labels from fields:
It is :
![image](https://user-images.githubusercontent.com/6230943/94018368-722b6100-fda8-11ea-9844-303c22c2c9fa.png)
Should be :
![image](https://user-images.githubusercontent.com/6230943/94018285-54f69280-fda8-11ea-8b0d-e1d7eda816e9.png)